sirepo/pkcli/job_cmd.py

```diff
@@ -104,8 +106,7 @@ def _do_compute(msg, template):
             r = p.poll()
             i = r is None
             if not i:
-                if os.WIFSIGNALED(r):
-                    pkdp('\n\n\n\n --------- \n RET STATUS INFO: {} \n info:{}\n ------- \n\n\n\n', r, os.WTERMSIG(r))
+                if os.WIFSIGNALED(r) and abs(r) == signal.SIGKILL:
```

Comment:
`subprocess` behaves differently than `os`. `os.WIFSIGNALED` has [already been called](https://github.com/python/cpython/blob/606e496dd6e2ace298532da200169124c26ae0f2/Lib/subprocess.py#L1851). Therefore, the test should be:

```py
if r == -signal.SIGKILL:
```

We know at this point `r` is `None`. By converting the constant, the test is exactly what we want: Was it KILLed?

The documentation in subprocess could be clearer. Something bothered me about the way it was written. It is technically correct, but a bit of help for the user (note: this is not an exit code).